### PR TITLE
Update the readme to contain info on other projects

### LINKED ISSUE #1538: Add info about other projects to the NUnit Readme.md
See issue nunit/nunit3-vs-adapter#174

We should have a section of the readme that lists other nunit projects and points to the repos. This may allay confusion on the part of users.

### PR DESCRIPTION
**Pull request by rprouse**
Fixes #1538 

I based this on @ChrisMaddock's suggested sections in the issue. I am open to changing what is included and the order, but it is probably easier to discuss with something concrete to look at.

The format I have selected is,

- Project Title
- Build related badges
- Social badges (I added gitter, should we keep it?)
- Short project blurb
- Download info
- Documentation links
- Contribution info
- License
- Contributors
- Links to other project repositories (I included most, but not all repositories, this could be pruned)

Once we settle on a standard format, we can use the same format for the other repositories, or at least the main ones.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,77 @@
...
+[![AppVeyor Build status](https://ci.appveyor.com/api/projects/status/3xfkxtnkrts1x06q/branch/master?svg=true)](https://ci.appveyor.com/project/CharliePoole/nunit/branch/master) [![Travis Build Status](https://travis-ci.org/nunit/nunit.svg?branch=master)](https://travis-ci.org/nunit/nunit) [![NuGet Version and Downloads count](https://buildstats.info/nuget/NUnit)](https://www.nuget.org/packages/NUnit)
 
-[![Follow NUnit](https://img.shields.io/twitter/follow/nunit.svg?style=social)](https://twitter.com/nunit)
+[![Follow NUnit](https://img.shields.io/twitter/follow/nunit.svg?style=social)](https://twitter.com/nunit) [![Gitter](https://img.shields.io/gitter/room/nwjs/nw.js.svg)](https://gitter.im/nunit/nunit) [![nunit-discuss Google Groups](https://img.shields.io/badge/mailing%20list-nunit--discuss-blue.svg)](https://groups.google.com/forum/m/#!forum/nunit-discuss)
```
Comment (ChrisMaddock) [inline README.md]: The Google Groups is a mobile link, should probably be https://groups.google.com/forum/#!forum/nunit-discuss instead? I'm definitely a fan of putting the gitter in, but I think we all knew that anyway! ;-)

File: README.md [hunk 2]
```diff
@@ -1,17 +1,77 @@
...
+## Documentation ##
+
+Documentation for all NUnit projects are hosted on GitHub at [https://github.com/nunit/docs/wiki](https://github.com/nunit/docs/wiki).
+
+- [NUnit Documentation](https://github.com/nunit/docs/wiki/NUnit-Documentation)
+- [Installation](https://github.com/nunit/docs/wiki/Installation)
+- [Release Notes](https://github.com/nunit/docs/wiki/Release-Notes)
+
+## Contributing ##
+
```
Comment (ChrisMaddock) [inline README.md]: Very happy we've added this section! 🎉

File: README.md [hunk 3]
```diff
@@ -1,17 +1,77 @@
...
 
 ## License ##
 
-NUnit is Open Source software and NUnit 3.0 is released under the [MIT license](http://www.nunit.org/nuget/nunit3-license.txt). Earlier releases used the [NUnit license](http://www.nunit.org/nuget/license.html). Both of these licenses allow the use of NUnit in free and commercial applications and libraries without restrictions.
+NUnit is Open Source software and NUnit 3 is released under the [MIT license](https://github.com/nunit/docs/wiki/License). Earlier releases used the [NUnit license](http://www.nunit.org/nuget/license.html). Both of these licenses allow the use of NUnit in free and commercial applications and libraries without restrictions.
 
 ## Contributors ##
 
 NUnit 3.0 was created by [Charlie Poole](https://github.com/CharliePoole), [Rob Prouse](https://github.com/rprouse), [Simone Busoli](https://github.com/simoneb), [Neil Colvin](https://github.con/oznetmaster) and numerous community contributors. A complete list of contributors since NUnit migrated to GitHub can be [found on GitHub](https://github.com/nunit/nunit/graphs/contributors).
```
Comment (ChrisMaddock) [inline README.md]: How about making this the second half of the "Contributing" section? Having separate "Contributing" and "Contributors" sections feels a little wordy.

File: README.md [hunk 4]
```diff
@@ -1,17 +1,77 @@
...
+NUnit is made up of several projects. When reporting issues, please try to report issues in the correct project.
+
+### Core Projects ###
+
+- [NUnit Test Framework](https://github.com/nunit/nunit) - The test framework used to write NUnit tests (this repository)
+- [NUnit Console and Engine](https://github.com/nunit/nunit-console) - Runs unit tests from the command line and provides the engine that is used by other test runners to run NUnit tests
+
+### Visual Studio Extensions ###
+
+- [NUnit 3 Visual Studio Adapater](https://github.com/nunit/nunit3-vs-adapter) - Visual Studio adapter for running NUnit 3 tests in Visual Studio and in VSTS/TFS builds
```
Comment (ChrisMaddock) [inline README.md]: "Adapater"

File: README.md [hunk 5]
```diff
@@ -1,17 +1,77 @@
...
+
+- [NUnit GUI](https://github.com/nunit/nunit-gui) - Runs NUnit 3 tests in a GUI based runner
+- [NUnit Xamarin Runner](https://github.com/nunit/nunit.xamarin) - Runs NUnit 3 tests on mobile devices using the Xamarin framework
+- [NUnit .NET Core Runner](https://github.com/nunit/dotnet-test-nunit) - Runs NUnit tests compiled against .NET Core
+- [NUnit Installer](https://github.com/nunit/nunit-distribution) - Creates the installer for the NUnit Console, Engine and Extensions
+- [NUnit Project Editor](https://github.com/nunit/nunit-project-editor) - Edits the NUnit Project format
+
+### NUnit Engine Extensions ###
+
+- [NUnit 2 Driver](https://github.com/nunit/nunit-v2-framework-driver) - Allows NUnit 3 to run NUnit 2 tests
```
Comment (ChrisMaddock) [inline README.md]: Allows NUnit 3 _engine_ to run NUnit 2 tests, perhaps? Possibly a bit persnickety, sorry!

File: README.md [hunk 6]
```diff
@@ -1,17 +1,77 @@
...
+- [NUnit 2 Result Writer](https://github.com/nunit/nunit-v2-result-writer) - Writes test results in the legacy NUnit 2 format
+- [Visual Studio Project Loader](https://github.com/nunit/vs-project-loader) - Loads and parses Visual Studio projects and solutions
+- [TeamCity Event Listenter](https://github.com/nunit/teamcity-event-listener) - Allows you to run and integrate NUnit tests into TeamCity
+
```
Comment (ChrisMaddock) [inline README.md]: NUnit project loader should probably be here as well.

File: README.md [hunk 7]
```diff
@@ -1,17 +1,77 @@
...
+### Samples ###
+
+- [C# NUnit Test Samples](https://github.com/nunit/nunit-csharp-samples)
+- [Visual Basic NUnit Test Samples](https://github.com/nunit/nunit-vb-samples)
+- [F# NUnit Test Samples](https://github.com/nunit/nunit-fsharp-samples)
+
+### Older Projects ###
+
+- [NUnit 2 Visual Studio Adapater](https://github.com/nunit/nunit-vs-adapter) - Visual Studio adapter for running older NUnit 2.x tests in Visual Studio and in VSTS/TFS builds
```
Comment (ChrisMaddock) [inline README.md]: Should the NUnit 2 adapter be up with the other Visual Studio extensions, whilst it's still being supported? That would then make this section solely 'legacy' projects.

File: README.md [hunk 8]
```diff
@@ -1,17 +1,77 @@
...
+- [NUnit 2 Result Writer](https://github.com/nunit/nunit-v2-result-writer) - Writes test results in the legacy NUnit 2 format
+- [Visual Studio Project Loader](https://github.com/nunit/vs-project-loader) - Loads and parses Visual Studio projects and solutions
+- [TeamCity Event Listenter](https://github.com/nunit/teamcity-event-listener) - Allows you to run and integrate NUnit tests into TeamCity
+
+### Samples ###
+
+- [C# NUnit Test Samples](https://github.com/nunit/nunit-csharp-samples)
+- [Visual Basic NUnit Test Samples](https://github.com/nunit/nunit-vb-samples)
+- [F# NUnit Test Samples](https://github.com/nunit/nunit-fsharp-samples)
+
```
Comment (ChrisMaddock) [inline README.md]: This is missing C++. I wonder if we actually want these and the 'older projects' section - I guess it depends how definitive a list we want this to be. An alternative suggestion (or possibly aswell), how about adding a link to https://github.com/nunit/docs/wiki/Samples in the documentation section above? Thought process: code samples are good 'getting started' content.

File: README.md [hunk 9]
```diff
@@ -1,17 +1,77 @@
...
+## NUnit Projects ##
+
+NUnit is made up of several projects. When reporting issues, please try to report issues in the correct project.
+
+### Core Projects ###
+
+- [NUnit Test Framework](https://github.com/nunit/nunit) - The test framework used to write NUnit tests (this repository)
+- [NUnit Console and Engine](https://github.com/nunit/nunit-console) - Runs unit tests from the command line and provides the engine that is used by other test runners to run NUnit tests
+
```
Comment (ChrisMaddock) [inline README.md]: Is it worth adding the docs repo under core-projects. Thought process: will people will use this list to find the right place to report an issue?

File: README.md (hunk 8 above)
Comment (ChrisMaddock) [inline README.md]: @rprouse - Sorry, I see you already asked the question on whether we want to prune the repo or not. In which case: my vote would be to cut the 'older projects' (with the NUnit 2 VS adapter moved up), and replace the sample repo's as above. 😄

**Comment by rprouse** [comment]
I have made most of the changes suggested including a TOC. It is a bit shorter and cleaner now, thanks @ChrisMaddock

**Pull request merged**